installing/installing_gcp/preparing-to-install-on-gcp.adoc

```diff
@@ -0,0 +1,48 @@
+[id="preparing-to-install-on-gcp"]
+= Preparing to install on GCP
+include::modules/common-attributes.adoc[]
+:context: preparing-to-install-on-gcp
+
+toc::[]
+
+[id="preparing-to-install-on-gcp-prerequisites"]
+== Prerequisites
+
+* You reviewed details about the xref:../../architecture/architecture-installation.adoc#architecture-installation[{product-title} installation and update] processes.
+
+* You read the documentation on xref:../../installing/installing-preparing.adoc#installing-preparing[selecting a cluster installation method and preparing it for users].
+
+[id="requirements-for-installing-ocp-on-gcp"]
+== Requirements for installing {product-title} on GCP
+
+Before installing {product-title} on Google Cloud Platform (GCP), you must create a service account and configure a GCP project. See xref:../../installing/installing_gcp/installing-gcp-account.adoc#installation-gcp-project_installing-gcp-account[Configuring a GCP Project] for details about creating a project, enabling API services, configuring DNS, GCP account limits, and supported GCP regions.
+
+If the cloud identity and access management (IAM) APIs are not accessible in your environment, or if you do not want to store an administrator-level credential secret in the `kube-system` namespace, see xref:../../installing/installing_gcp/manually-creating-iam-gcp.adoc#manually-creating-iam-gcp[Manually creating IAM for GCP] for other options.
+
+[id="choosing-an-method-to-install-ocp-on-gcp"]
+== Choosing a method to install {product-title} on GCP
+
+You can install {product-title} on GCP by using one of the following deployment methods.
+
+* **xref:../../installing/installing_gcp/installing-gcp-default.adoc#installing-gcp-default[Installing a cluster quickly on GCP]**: You can install {product-title} on GCP infrastructure that is provisioned by the {product-title} installation program. You can install a cluster quickly by using the default configuration options.
+
+* **xref:../../installing/installing_gcp/installing-gcp-customizations.adoc#installing-gcp-customizations[Installing a customized cluster on GCP]**: You can install a customized cluster on GCP infrastructure that the installation program provisions. The installation program allows for some customization to be applied at the installation stage. Many other customization options are available xref:../../post_installation_configuration/cluster-tasks.adoc#post-install-cluster-tasks[post-installation].
+
+* **xref:../../installing/installing_gcp/installing-gcp-network-customizations.adoc#installing-gcp-network-customizations[Installing a cluster on GCP with network customizations]**: You can customize your {product-title} network configuration during installation, so that your cluster can coexist with your existing IP address allocations and adhere to your network requirements.
```

Comment:
Is this one also IPI?

Reply:
Yes, both these steps are IPI. But I had to read awhile to be sure. The term "installation program" doesn't show up for a little bit in the referenced text.

Reply:
My understanding is that the networking customisations that are outlined in this assembly could be applied to IPI or UPI installations. Although the assembly does provide an IPI end-to-end example, I think the purpose of the assembly is to highlight the network customisations that are possible. My view is that if we state that this is for IPI specifically in this high level bullet point, it might detract from the fact that you can apply the customisations in different scenarios.

Reply:
The assembly provides an IPI end-to-end example. But, could some of the networking parameters be relevant also to UPI scenarios?

Reply:
We have decided to split the bullet points for all of the "Preparing to install on..." assemblies into higher level bulleted items, one for IPI and one for UPI. As such, this can remain as is for now and in a further PR this week we will categorise this as an IPI sub-bullet point.